Yoast/javascript
# Add an exception check for plurals ending on -x

## Summary

This PR can be summarized in the following changelog entry:

* [yoasteo] Checks an exception list of plurals with -x suffix and stems them correctly.
* [Yoast SEO Premium] Improves keyphrase recognition when keyphrase contains a word which gets the -x suffix in plural (e.g., jeux).


## Test instructions

This PR can be tested by following these steps:

* Make sure all current specs are passing
* Add more words from the plurals on -x exception list to stemSpec and make sure the specs still pass
* In French Premium: Try using different forms of French words that get -x in plural in the keyphrase and in the title/text/meta description and see that those are recognized. E.g., "cheveux" and "cheveu" should both be recognised.


## Quality assurance


Fixes https://yoast.atlassian.net/jira/software/projects/LIN/boards/4?selectedIssue=LIN-409
